### PR TITLE
Add ignore_checksum_failure GUC to reference

### PR DESCRIPTION
**Pull request by litzell**

### CODE REVIEW

File: gpdb-doc/dita/ref_guide/config_params/guc-list.xml [hunk 1]
```diff
@@ -5507,6 +5508,42 @@
       </table>
     </body>
   </topic>
+  <topic id="ignore_checksum_failure">
+    <title>ignore_checksum_failure</title>
+    <body>
+      <p>Only has effect if <xref href="#data_checksums" format="dita"/> is enabled.</p>
+      <p>Detection of a checksum failure when reading a heap data page normally causes Greenplum
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/config_params/guc-list.xml]: Would really prefer this sentence in active voice:  "By default, if Greenplum Database detects ..."

File: gpdb-doc/dita/ref_guide/config_params/guc-list.xml [hunk 2]
```diff
@@ -5507,6 +5508,42 @@
...
+        Database to report an error, aborting the current transaction. Setting
+          <codeph>ignore_checksum_failure</codeph> to on causes the system to ignore the failure,
+        but still report a warning, and continue processing. This behavior may <b>cause crashes,
+          propagate or hide corruption, or lead to other serious problems</b>. However, it may allow
+        you to get past the error and retrieve undamaged tuples that might still be present in the
+        table if the block header is still sane. If the header is corrupt an error will be reported
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/config_params/guc-list.xml]: Need to replace "sane" with something else.  Uncorrupted or undamaged, maybe?  Consistent?

File: gpdb-doc/dita/ref_guide/config_params/guc-list.xml [hunk 3]
```diff
@@ -5507,6 +5508,42 @@
...
   </topic>
+  <topic id="ignore_checksum_failure">
+    <title>ignore_checksum_failure</title>
+    <body>
+      <p>Only has effect if <xref href="#data_checksums" format="dita"/> is enabled.</p>
+      <p>By default, when Greenplum Database detects a checksum failure when reading a heap data
+        page it reports an error, aborting the current transaction. Setting
+          <codeph>ignore_checksum_failure</codeph> to on causes the system to ignore the failure,
+        but still report a warning, and continue processing. This behavior may <b>cause crashes,
+          propagate or hide corruption, or lead to other serious problems</b>. However, it may allow
```
Comment (mkiyama) [inline gpdb-doc/dita/ref_guide/config_params/guc-list.xml]: I would reword some of this to put the the This behavior information in a Warning:

I'm not sure if setting to ON will cause crashes.
